ADR/adr-0007-change-management.md

```diff
@@ -0,0 +1,70 @@
+
+This change management process will go into effect when App Studio goes into a 24x7 support mode; this will likely correspond to its GA date.
+
+## Decision
+
+Incremental code changes that are fully tested by automated tests, which will not cause an outage or a significant change to functionality, will follow the normal code review process in their respective code repositories. 
+
+To deploy an infrastructure change or a new version of software to staging and production, a developer will make the required change to the infra-deployments repo or the App Interface repo.  Then the change must follow our normal code review process.
+
+### Normal code review process
+
+While a ticket (Jira ticket or Github issue) is In Progress, a developer will make their code changes and write automated tests in a Git feature branch.  The developer will raise a Github Pull Request (PR) when they need other team members to review their work. This review could be for work in progress or when the developer feels the work is completed. For a PR that reflects completed work, the ticket should be put into the Review stage to indicate that active development is complete.
+
+In Github, if the developer wants a code review but feels that their work is not ready to be deployed yet, they will add the "do-not-merge/work-in-progress" label to the PR. We also recommend adding "WIP" to the pull request title.
+
+The developer should choose one or two reviewers for their PR and add them to the PR directly. A PR without any reviewers assigned might never be reviewed. Reviewers should be chosen based on their background and familiarity with the code under review. It is also sometimes worthwhile to use a PR review to help another developer get up to speed with some code.
+
+If you have been asked to review another developer’s PR, it’s good practice to decide if you can fit that review into your current workload. Timely PR review helps keep our velocity high, and if it might take more than a couple of days for you to review the code, contact the PR author and let them know they might need to depend on another person.
+
+As a reviewer, you should move the review along by testing the PR yourself, and providing comments in Github for suggested changes. Typically we want to focus on potential security or privacy issues, test automation, maintainability factors and overall correctness of the code. Stylistic or preference-based changes should not be a focus of the review.
+
+We also require that all of our repositories be instrumented with CI test automation that runs when a PR is raised, and after each new commit. All CI checks must pass before performing a Merge to the main branch as this updates the staging environment and makes the code available for deployment into production.
+
+When the review is completed and tests have passed, the approver should Approve the PR in GitHub and the PR author will then Merge the code. 
```

Comment:
Good point.  I'll add that in.

Reply:
I added the things that would apply right at code review time.

Reply:
Actually, let's try to keep the Change Management process slim, because it's an auditable artifact.  We don't want auditors to get lost in the weeds of things that aren't specifically designed to meet the goals of Change Management.  I moved the rest into our Engineering Standards document.  We could possibly publish that here too.

Reply:
+1 to move the standards document to the book.